recipe/test_libtensorflow.sh

```diff
@@ -2,5 +2,5 @@
 
 set -euxo pipefail
 
-$CC -o test_c -L${PREFIX}/lib/ -ltensorflow -I${PREFIX}/include/ test_c.c
+${CC} -o test_c ${CFLAGS} ${LDFLAGS} -ltensorflow test_c.c
```

Comment:
This is expanding to:
```
/bin/x86_64-conda-linux-gnu-cc \
-o test_c \
-march=nocona -mtune=haswell -ftree-vectorize -fPIC \
-fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe \
-isystem /home/conda/feedstock_root/build_artifacts/tensorflow-split_1647631094566 \
/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/include -fdebug-prefix-map=/home/conda/feedstock_root/build_artifacts/tensorflow-split_1647631094566/test_tmp=/usr/local/src/conda/libtensorflow-2.8.0 \
-fdebug-prefix-map=/home/conda/feedstock_root/build_artifacts/tensorflow-split_1647631094566/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold=/usr/local/src/conda-prefix \
-Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,--disable-new-dtags -Wl,--gc-sections \
-Wl,--allow-shlib-undefined \
-Wl,-rpath,/home/conda/feedstock_root/build_artifacts/tensorflow-split_1647631094566/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib -Wl,-rpath-link,/home/conda/feedstock_root/build_artifacts/tensorflow-split_1647631094566/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib \
-L/home/conda/feedstock_root/build_artifacts/tensorflow-split_1647631094566/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib \
-ltensorflow test_c.c
```

Seems to be correct.

Reply:
Somehow the order mattered locally.